src/config.js

```
@@ -0,0 +1,71 @@
+        errors.push('DISCORD_TOKEN is required in .env file');
+    }
+    
+    if (config.google.credentials && !require('fs').existsSync(config.google.credentials)) {
+        console.warn(`⚠️  Google credentials file not found: ${config.google.credentials}`);
+        console.warn('   Google Calendar features will be disabled.');
+        config.google.credentials = null;
+    }
+    
+    if (errors.length > 0) {
+        console.error('\n❌ Configuration Errors:\n');
+        errors.forEach(err => console.error(`  - ${err}`));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)